test-suite/unit-tests/tests/economics.test.js

```diff
@@ -39,5 +39,5 @@ test("prepaidGas", async t => {
 test("usedGas", async t => {
   const {root, contract} = t.context.accounts
   const res = await root.call(contract, "doUsedGas", {})
-  t.deepEqual(res, 274214078913)
+  t.deepEqual(res, 274446183237);
```

Comment:
Is this flakey? I thought philip changed this number in his final commit?

Reply:
This is more or less going to change with every release of Grain. We should really be testing for a reasonable range.